api/serializers/search_result.py

```diff
@@ -9,17 +9,23 @@ class SearchResultSerializer(serializers.Serializer):
     name = serializers.CharField(read_only=True)
     name_en = serializers.CharField(read_only=True)
 
+    # Plant
+    plantsynonym_set = serializers.SlugRelatedField(read_only=True, many=True, slug_field="name")
+
     # Ingredient
     description = serializers.CharField(read_only=True)
     observation = serializers.CharField(read_only=True)
+    ingredientsynonym_set = serializers.SlugRelatedField(read_only=True, many=True, slug_field="name")
 
     # Substance
     cas_number = serializers.CharField(read_only=True)
     einec_number = serializers.CharField(read_only=True)
     source = serializers.CharField(read_only=True)
+    substancesynonym_set = serializers.SlugRelatedField(read_only=True, many=True, slug_field="name")
 
     # Microorganism
     genre = serializers.CharField(read_only=True)
+    microorganismsynonym = serializers.SlugRelatedField(read_only=True, many=True, slug_field="name")
```

Comment:
Je pense qu'il manque le `_set`
```suggestion
    microorganismsynonym_set = serializers.SlugRelatedField(read_only=True, many=True, slug_field="name")
```